modules/rhel-images-aws.adoc

```diff
@@ -0,0 +1,29 @@
+[id="rhel-images-aws_{context}"]
```

Comment:
For modules, we try and provide a commented out section that lists the assemblies the module is included in (for reference and convenience). For example, the top of this module would look something like this:
```
// Module included in the following assemblies:
//
// * machine_management/adding-rhel-compute.adoc

[id="rhel-images-aws_{context}"]
= Listing latest available RHEL images on AWS
...
```

Reply:
Fixed; I didn't realize this was manually added. Thanks for that.